### PR TITLE
chore(ci): run replication tests on arm

### PR DESCRIPTION
**Pull request by kostasrim**
* allow running replication tests on arm

### CODE REVIEW

**Review by romange — COMMENTED** [review]
what's the reason for this change?

**Comment by kostasrim** [comment]
@romange Adi wanted to run replication tests on arm as well :) 

I think it makes sense :)

File: .github/actions/regression-tests/action.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ runs:
 
     - name: Run PyTests replication test
       id: second
-      if: ${{ inputs.run-only-on-ubuntu-latest == 'true' || (inputs.run-only-on-ubuntu-latest == 'false' && matrix.runner == 'ubuntu-latest') }}
+      if: ${{ (inputs.run-only-on-ubuntu-latest == 'true' && matrix.runner == 'ubuntu-latest') || (inputs.run-only-on-ubuntu-latest == 'false' }}
```
Comment (adiholden) [inline .github/actions/regression-tests/action.yml:55]: why do we still need this if ? cant we just remove it?

Comment (kostasrim) [reply .github/actions/regression-tests/action.yml:55]: We can, I think we didn't want to run them on release. Let me check :)

**Pull request merged**